GUI_detect.py

```diff
@@ -198,10 +198,8 @@ def detect(save_img=False):
                             save_path += '.mp4'
                         vid_writer = cv2.VideoWriter(save_path, cv2.VideoWriter_fourcc(*'mp4v'), fps, (w, h))
                     vid_writer.write(im0)
-
-        if not flag:
-            flag = True  # start GUI
-            time.sleep(2)  # let GUI start up
+        
+        detect_ready.set()
```

Comment:
- Reverted the time.sleep to the original way which hopefully fixes the issue
- Since the board seems to work without it, I can't verify if it works properly in your machine. 
- I have no idea why the buttons dont work since they seem to function fine on my end. I tried to do some debugging and find if it was a sensor or CV queue blocking issue, but I couldn't find anything.
- Ill leave it unresolved for now, but once everything looks good on ur side, feel free to mark it as resolved and merge.

Reply:
BTW I also sped up the video speed so that it runs similar to real-time on the board, lemme know if this causes any issues on your side as well